responsiveness-homework-starter-code/css/index.css

```diff
@@ -33,16 +35,38 @@ header, .experience {
 }
 .containertext
 {
-	width: 60%;
+
 	height: auto;
 	border: 1px solid #E13A63;
 	display: inline-block;
+
 }
-.containerimg
+
+
+.containertext1
 {
```

Comment:
This is not a very descriptive class name. Also, you apply this class to multiple elements, whilst the name implies it just refers to one element. Be more specific with your class names when possible.